ctyparser/bigcty.py

```diff
@@ -172,7 +173,9 @@ def update(self) -> bool:
 
             with tempfile.TemporaryDirectory() as temp:
                 path = pathlib.PurePath(temp)
-                dl_url = f'http://www.country-files.com/bigcty/download/{update_date[:4]}/bigcty-{update_date}.zip'  # TODO: Issue #10
+                page = session.get(update_url)
+                tree = html.fromstring(page.content)
+                dl_url = tree.xpath("//a[contains(@href,'zip')]/@href")[0]
```

Comment:
what if `tree.xpath()` doesn't find anything? would `[0]` `IndexError`? we should handle that, perhaps

Reply:
Agreed